dmcp/fix.py

```diff
@@ -12,45 +13,59 @@
 from cvxpy.constraints.zero import Zero
 from cvxpy.constraints.psd import PSD
 
+from cvxpy.constraints.nonpos import NonNeg
```

Comment:
can you merge these "import NonNeg, NonPos"?

Reply:
sounds good